common/util/print_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2022-2023 Nutanix, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import "testing"
+
+func TestToString(t *testing.T) {
```

Comment:
Nit: Rename to intToString? It's possible to convert other objects to their string representation, up to you.

Reply:
This is a test file, the underlying function is called "ToString", by go test convention we just append the name of the function to be tested with a `Test` prefix.

Reply:
I know I was just convering if it would be better we name the core function to "inToString" but now that I think about it "ToString" is descriptive enough and is the convention in other languages.